test/extended/testdata/verifyservice-pipeline-template.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: nodejs-example
+metadata:
+  annotations:
+    iconClass: icon-jenkins
+    tags: instant-app,jenkins
+  name: nodejs-example
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      description: Normal Service to be tested
+    name: nodejs-example
+  spec:
+    ports:
+    - name: web
+      port: 8080
+      targetPort: 8080
+    selector:
+      name: nodejs-example
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      description: Headless Service to be tested
+    name: nodejs-example-headless
+  spec:
+    clusterIP: None
+    ports:
+    - port: 8080
+      targetPort: 8080
+    selector:
+      name: nodejs-example
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    annotations:
+      description: Defines how to deploy the application server
+      template.alpha.openshift.io/wait-for-ready: "true"
+    name: nodejs-example
+  spec:
+    replicas: 2
+    selector:
+      name: nodejs-example
+    strategy:
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          name: nodejs-example
+        name: nodejs-example
+      spec:
+        containers:
+        - env: []
+          image: ' '
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 8080
+            initialDelaySeconds: 30
+            timeoutSeconds: 3
+          name: nodejs-example
+          ports:
+          - containerPort: 8080
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 8080
+            initialDelaySeconds: 3
+            timeoutSeconds: 3
+    triggers:
+    - imageChangeParams:
+        automatic: true
```

Comment:
image change trigger is unnecessary ... remove it, just leave the config change trigger

Reply:
Removed the image trigger